packages/design-system/src/components/Icon/Icon.js

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+
+import { IconSize, IconColor } from './constants';
+import { generateIcon } from './helpers';
+
+export const Icon = props => {
+  const { icon, iconSize, iconColor, className, ...restProps } = props;
+
+  const size = IconSize[iconSize];
+  const color = IconColor[iconColor];
+  const GeneratedIcon = generateIcon(icon, size, color);
+
+  return (
+    <span {...restProps} className={className}>
+      {GeneratedIcon}
+    </span>
+  );
+};
+
+Icon.propTypes = {
+  /**
+   * Icon passed in svg format
+   */
+  icon: PropTypes.func.isRequired,
```

Comment:
`<Icon icon={required('///')} />` vs `<Icon source={required('///')} />`
wdyt?

Reply:
yeah, I think `source` is the better name here than `icon` 👍